src/AppInstallerCommonCore/AppInstallerLogging.cpp

```diff
@@ -130,6 +130,21 @@ namespace AppInstaller::Logging
         }
     }
 
+    std::string_view DiagnosticLogger::GetLogFileNamePrefix()
```

Comment:
And if I did feel it was necessary to change the prefix of the installer log file, I would want to use something cleaner than this.  If there is no file log, then installer log files have no prefix?

I think that your last two commits should be reverted except for your one change that was unrelated.

If you do want to force the installer log file to have the same prefix, then it needs to be done another way.  The first thing that comes to mind is storing the default as a member on `DiagnosticLogger`.

Reply:
Undoing the change.